lib/active_record_doctor/rake/task.rb

```diff
@@ -69,7 +69,24 @@ def runner
 
       def config
         @config ||= begin
-          path = config_path && File.exist?(config_path) ? config_path : nil
+          path = nil
+
+          if config_path
+            if File.exist?(config_path)
+              path = config_path
+            elsif config_path.to_s.end_with?(".rb")
+              config_path_without_extension = config_path.to_s.gsub(/\.rb\z/, "")
+              if File.exist?(config_path_without_extension)
+                warn <<~WARN.squish
+                  [DEPRECATED] The default configuration
+                  file #{config_path_without_extension} is deprecated.
+                  Use #{config_path} instead.
+                WARN
+                path = config_path_without_extension
+              end
+            end
+          end
```

Comment:
This code would affect all configuration files, not only to default one. That logic should be applied only to cases where the user did not specify a configuration file and is relying on the default value instead. If the user explicitly requested an extensionless file we should not run this logic at all.

Inspired by your PR I open [my own](https://github.com/gregnavis/active_record_doctor/pull/166/files) and would like to ask you for a quick review. Thanks!